deploy/examples/import-external-cluster.sh

```diff
@@ -4,7 +4,7 @@ set -e
 ##############
 # VARIABLES #
 #############
-NAMESPACE=${NAMESPACE:="rook-ceph-external"}
+NAMESPACE=${NAMESPACE:="rook-ceph"}
```

Comment:
We need to revert this? The default should still be `rook-ceph-external`, but the doc referring to this script should just set it to `rook-ceph`, right?

Reply:
This was changed yesterday after @parth-gr review. It will allow us to use import script with the same namespace.  I already mentioned I. The doc if we have a different namespace , can change the namespace variable. It also make sense .  @parth-gr I think you have the same opinion.

Reply:
Also user mostly set the --namsepace flag whereever they want to run there ceph cluster, it is just the default value if flag is not set.

Reply:
@parth-gr do I need to revert the default value, as user use the --namespace flag.

Reply:
Ok, let's go with this. I just wanted to make sure existing users wouldn't be broken by change in defaults.